File: API/Entities/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using API.Extensions;

namespace API.Entities
{
    public class Student : AppUser
    {
        public string Name { get; set; }
        public string Surname { get; set; }
        public string StdNum { get; set; }
        public string Email { get; set; }
        public string Country { get; set; }        
        public string Gender { get; set; }
        public DateTime DateOfBirth { get; set; }
        public DateTime Created { get; set; }=DateTime.Now;
        public DateTime LastActive { get; set; }=DateTime.Now;
        public string Faculty { get; set; }
        public ICollection<StudentPhoto> Photos { get; set; }        
        public ICollection<Course> Courses { get; set; }
        public int GetAge(){ 
            return DateOfBirth.CalculateAge();
        }
    }
}

File: client/src/app/_models/student.ts
import { Course } from "./course";

    export interface Student {
        id: number;
        username: string;
        name: string;
        surname: string;
        stdNum: string;
        photoUrl: string;
        email: string;
        country: string;
        gender: string;
        age: number;
        created: Date;
        lastActive: Date;
        faculty: string;
        photos: StdPhoto[];
        courses: Course[];
        homeworks: Homework[];
    }
    export interface StdPhoto {
        id: number;
        url: string;
        isMain: boolean;
    }
    export interface Homework {
        id: number;
        url: string;
        fileName: string;
        homeworkName: string;
    }

  

   

